lib/components/chat_history/message.dart

```diff
@@ -28,6 +28,14 @@ class ChatHistoryMessage extends StatelessWidget {
         if (layoutModel.isInteractionLockable && layoutModel.locked) {
           return child;
         }
+        final userModel = Provider.of<UserModel>(context, listen: false);
+        final loginChannel = userModel.userChannel!.displayName;
```

Comment:
Can we compare by channel id? The format for `TwitchMessageModel.channelId` is `$provider:$channelId`. This is more robust when we add other providers.

Reply:
I added `channelId` to `TwitchMessageModel`, and I changed it so its compared by `channelId`.